pasqal_cloud/batch.py

```diff
@@ -181,23 +183,23 @@ def retry(self, job: Job, wait: bool = False) -> None:
         except JobCreationError as e:
             raise JobRetryError from e
 
-    def declare_complete(self, wait: bool = False, fetch_results: bool = False) -> None:
-        """Declare to PCS that the batch is complete and returns an updated
+    def close(self, wait: bool = False, fetch_results: bool = False) -> None:
```

Comment:
breaking change ?

Reply:
I didn't notice that @MatthieuMoreau0 should I refrain from renaming these methods since it's a breaking change?
If I understand correctly we're keeping complete and open together so that users don't have to update their scripts. This would probably defeat that purpose.

Reply:
imo we should:
- add a `close` method like you did here
- keep the `declare_complete` method
  - make it raise a deprecation warning for the user to invite them to use `close` instead
  - use the `close` method in its body so that we do not duplicate the logic